Celeste.Mod.mm/Patches/Autotiler.cs

```diff
@@ -72,8 +72,10 @@ public patch_Autotiler(string filename)
             if (xml.HasAttr("soundPath") && xml.HasAttr("sound")) { // Could accommodate for no sound attr, but requiring it should improve clarity on user's end 
                 SurfaceIndex.TileToIndex[xml.AttrChar("id")] = xml.AttrInt("sound");
                 patch_SurfaceIndex.IndexToCustomPath[xml.AttrInt("sound")] = (xml.Attr("soundPath").StartsWith("event:/") ? "" : "event:/") + xml.Attr("soundPath");
-            } else if (!SurfaceIndex.TileToIndex.ContainsKey(xml.AttrChar("id")) || xml.AttrChar("id") == 'o') { // Backwards-compat: some existing mods use 'o' and overwrite its sound
-                SurfaceIndex.TileToIndex[xml.AttrChar("id")] = xml.HasAttr("sound") ? xml.AttrInt("sound") : 8; // 8 as fallback instead of 0 to match vanilla's default index
+            } else if (xml.HasAttr("sound")) {
+                SurfaceIndex.TileToIndex[xml.AttrChar("id")] = xml.AttrInt("sound");
+            } else if (!SurfaceIndex.TileToIndex.ContainsKey(xml.AttrChar("id"))) {
+                SurfaceIndex.TileToIndex[xml.AttrChar("id")] = SurfaceIndex.Brick; // 8 as fallback instead of 0 to match vanilla's default index
```

Comment:
Re: discord discussion, 0 or -1 (no sound) would be a better default, since that makes it consistent with current solid tile behavior. Probably 0 (assuming it doesn't crash) since that's what's on the wiki + in the FMOD project lol. Rest of it looks good though!

Reply:
Falling back to 0. -1 is used in contexts where there are no tiles below the player, as far as I can tell.